detox/src/devices/drivers/AndroidDriver.js

```diff
@@ -125,6 +130,24 @@ class AndroidDriver extends DeviceDriverBase {
     // Other payload content types are not yet supported.
   }
 
+  async waitUntilReady() {
+    let intervalId;
+    try {
+        await Promise.race([
+          super.waitUntilReady(),
+          new Promise((resolve, reject) => {
+            intervalId = setInterval(() => {
```

Comment:
Well I want to poll the state of the instrumentation process so it's either using setInterval or calling setTimeout repeatedly. I suppose the former is cleaner.
But on 2nd thought, no real reason for this to be polled, since we control both ends of this process. I'll change it to be explicit! thanks!